articles/azure/azs-how-add-data-managed-portal.md

```diff
@@ -0,0 +1,113 @@
+---
```

Comment:
Actual file name is different to the one used in the metadata. Suggest changing the actual file name to:
azs-how-add-managed-disks-portal.md